### PR TITLE
chore: fixes generation of FOSS license overview with other plugin

### PR DESCRIPTION
**Pull request by jekutzsche**
The plugin npm-license-crawler caused an error in the build and was
therefore replaced by license-checker.

Refs iris-connect/iris-backlog#107

### CODE REVIEW

**Review by faversente — APPROVED** [review]
Looks good. If it works locally, please merge.

File: iris-client-fe/package.json [hunk 1]
```diff
@@ -8,7 +8,7 @@
     "build": "vue-cli-service build",
     "test:unit": "vue-cli-service test:unit",
     "lint": "vue-cli-service lint",
-    "generate-licenses": "npm-license-crawler --dependencies --csv licenses.csv",
+    "generate-licenses": "license-checker --csv --out licenses.csv --dependencies",
```
Comment (mad-nuts) [inline iris-client-fe/package.json:11]: license-checker --csv --out licenses.csv **--dependencies**
Should we remove the depencencies restricton to include the dev dependencies as well?

**Review by mad-nuts — CHANGES_REQUESTED** [review]

**Review by mad-nuts — APPROVED** [review]

File: iris-client-fe/package.json (hunk 1 above)
Comment (faversente) [reply iris-client-fe/package.json:11]: It actually looks like the flag is not available for license-checker:

https://www.npmjs.com/package/license-checker

Comment (mad-nuts) [reply iris-client-fe/package.json:11]: If the --dependencies parameter is not present, the checker will include all licenses

Comment (jekutzsche) [reply iris-client-fe/package.json:11]: Yes, `--dependencies` I actually wanted to delete, but obviously didn't. `--dependencies` hasn't an effect.

**Review by faversente — APPROVED** [review]

**Pull request merged**